docs/.vuepress/config.js

```diff
@@ -304,33 +305,33 @@ module.exports = {
           title: 'Configuration',
           children: [
             'configuration/',
```

Comment:
Left this at the top since its the same as the title block so I think this looks better but its not 100% alphabetically sorted, same goes for `developers` section.